File: For Fun/Assignment 1/C# version/BMI.cs
using System;

class BMI
{
	public static void Main(string[] args)
	{
		Console.Write("Give your length in meters: ");
		double length = Convert.ToDouble(Console.ReadLine());
		Console.Write("Give your weight in kilograms: ");
		double weight = Convert.ToDouble(Console.ReadLine());
		Console.WriteLine("You BMI is:  " + Math.Round(weight / (length * length)));
	}
}

File: sb223ce_assign1/BMI.java
package sb223ce_assign1;
import java.util.Scanner;

public class BMI {

	public static void main(String[] args) {
	Scanner scan = new Scanner(System.in);
	
	//  user's instruction 
	System.out.print("Give your length in meters: ");
	double length;
	length = scan.nextDouble();
	
	System.out.print("Give your weight in kilograms: ");
	double weight;
	weight = scan.nextDouble();
	
    int BMI;
	BMI= (int)(Math.round(weight/Math.pow(length,2)));  
	//first round off to nearest number and change from double to integer
	
	System.out.println("Your BMI is: "+BMI);
	scan.close();
	}

}
